neurons/validator.py

```diff
@@ -120,11 +121,17 @@ async def send_interval_stats(self):
         if metrics and len(metrics) > 0:
             bt.logging.info(f'Total submission to export: {len(metrics)}')
             chunk_metrics = split_chunks(list(metrics), 15000)
-            async for metrics in chunk_metrics:
-                self.send_interval_data(miner_data=metrics)
-                bt.logging.info(f'chunk submissions processed {len(metrics)}')
-                await asyncio.sleep(4)
-            self.event_provider.mark_submissions_as_exported()
+            try:
+                async for metrics in chunk_metrics:
+                    intervals = [metric[3] for metric in metrics]
+                    if not self.send_interval_data(miner_data=metrics):
+                        bt.logging.error('Interrupting the stream, due to backend error..')
+                        return
+                    bt.logging.info(f'chunk submissions exported {len(metrics)} intervals: {set(intervals)}')
+                    await asyncio.sleep(4)
```

Comment:
this looks a bit weird, why do we have sleep in asyncio?

Reply:
thats the purpose of chunks, to not spam backend  with 100k records at once, like a cooldown.

Reply:
ok, is 100k a lot or very little? and why 4 seconds and not 1 or 10...or simply check if the backend is available?